lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = opts['database']['type'] || 'mongodb'
```

Comment:
You are using class variables here "@@" instead of instance variables "@".

Reference: https://www.tutorialspoint.com/ruby/ruby_variables.htm

I believe you want instance variables here.